### ISSUE TITLE
Please use the Google form for reporting typing issues

### ISSUE THREAD

**Issue by hiroyuki-komatsu** [google/mozc#1056]
We've received numerous reports regarding typing issues, and we truly appreciate your valuable feedback.

To streamline the process from issue reporting to resolution, we've created a dedicated Google Form for typing issues, replacing the previous use of GitHub issues. We believe this will simplify the reporting process for both reporters and maintainers, as well as better organize typing-related issues from other concerns.

Form for reporting typing issues:
* https://forms.gle/oXgwbqzvSyVHNBSc7

The filled reports are available at:
* [Typing issue for Mozc (Responses)](https://docs.google.com/spreadsheets/d/1AAK8wiTfIhj339Rb98lIRwgGfHndVz-jFXFNXZNQyeo/pubhtml?gid=649291758&single=true)

If this new form doesn't work well, we would update the process again.
Thank you for your cooperation.

**Comment by ghost** [comment]
Reference欄において複数のURLを列挙したい場合はどうすればよいのでしょうか。

**Comment by hiroyuki-komatsu** [comment]
For multiple URLs, please use the Context field.
I have updated the descriptions in the form.

Thank you,

**Comment by ghost** [comment]
ご対応ありがとうございます。了解いたしました。

**Comment by ghost** [comment]
通常の変換候補ではなく、単漢字変換の候補に関するイシューもこのフォームに送信してよいのでしょうか。それとも、単漢字変換については従前通りGitHubにイシューを立てるべきなのでしょうか。

**Comment by hiroyuki-komatsu** [comment]
Yes, please use this form for symbols including single kanji characters too.
I added "Symbol" to the category.

Thank you,

**Comment by ghost** [comment]
了解いたしました。ありがとうございます。

**Comment by tats-u** [comment]
I found inconsistent readings for words that end with "型付け".

```
rg --engine auto '型付け' .\src\data\dictionary_oss\
.\src\data\dictionary_oss\dictionary04.txt
51371:かたつけ  1841    1841    6098    型付け

.\src\data\dictionary_oss\dictionary01.txt
79594:せいてきかたつけげんご    1931    1851    7389    静的型付け言語
80051:てもみかたつけ    1876    1841    5392    手もみ型付け
80217:どうてきかたつけげんご    1931    1851    7369    動的型付け言語

.\src\data\dictionary_oss\dictionary02.txt
85219:どうてきかたずけ  1929    1929    7219    動的型付け
85220:どうてきかたづけ  1929    1929    6068    動的型付け
```

"動的型付け" is registered only as "–づ(ず)け" but "型付け" and "–型付け言語" are registered only as "–つけ".

- https://www.google.co.jp/search?q=("動的型付け"+OR+"静的型付け")+-wikipedia+("かたつけ"+OR+"かたづけ")
- https://ja.wikipedia.org/wiki/%E9%9D%99%E7%9A%84%E5%9E%8B%E4%BB%98%E3%81%91 (no references)
- https://ja.wikipedia.org/wiki/%E5%8B%95%E7%9A%84%E5%9E%8B%E4%BB%98%E3%81%91 (no references)

Should we register missing entries using aux_dictionary.txt or words.tsv?

**Comment by hiroyuki-komatsu** [comment]
@tats-u For questions about aux_dictionary.txt or words.tsv, please create a new discussion. (I will hide your question and this my answer in a few days).
